packages/hoppscotch-app/locales/hi.json

```diff
@@ -0,0 +1,309 @@
+    "create_secret_gist": " गुप्त सार बनाएँ",
+    "gist_created": " जिस्ट बनाया",
+    "require_github": " गुप्त सार बनाने के लिए GitHub के साथ लॉगिन करें",
+    "title": " निर्यात करना"
+  },
+  "folder": {
+    "created": " फोल्डर बनाया",
+    "edit": " फ़ोल्डर संपादित करें",
+    "invalid_name": " कृपया फ़ोल्डर के लिए एक नाम प्रदान करें",
+    "name_length_insufficient": " फ़ोल्डर का नाम कम से कम 3 वर्ण लंबा होना चाहिए",
+    "new": " नया फोल्डर",
+    "renamed": " फ़ोल्डर का नाम बदला"
+  },
+  "graphql": {
+    "mutations": " उत्परिवर्तन",
+    "schema": " योजना",
+    "subscriptions": " सदस्यता"
+  },
+  "header": {
+    "install_pwa": " एप्लिकेशन इंस्टॉल करें",
+    "login": " लॉग इन करें",
+    "save_workspace": " मेरा कार्यक्षेत्र सहेजें"
+  },
+  "helpers": {
+    "authorization": " जब आप अनुरोध भेजेंगे तो प्राधिकरण शीर्षलेख स्वतः उत्पन्न हो जाएगा।",
+    "generate_documentation_first": " पहले दस्तावेज़ तैयार करें",
+    "network_fail": " एपीआई समापन बिंदु तक पहुंचने में असमर्थ। अपने नेटवर्क कनेक्शन की जाँच करें या किसी भिन्न इंटरसेप्टर का चयन करें और पुनः प्रयास करें।",
+    "offline": " ऐसा लगता है कि आप ऑफ़लाइन हैं. हो सकता है कि इस कार्यस्थान का डेटा अप टू डेट न हो.",
+    "offline_short": " ऐसा लगता है कि आप ऑफ़लाइन हैं.",
+    "post_request_tests": " टेस्ट स्क्रिप्ट जावास्क्रिप्ट में लिखी जाती हैं, और प्रतिक्रिया प्राप्त होने के बाद चलाई जाती हैं।",
+    "pre_request_script": " पूर्व-अनुरोध स्क्रिप्ट जावास्क्रिप्ट में लिखी जाती हैं, और अनुरोध भेजे जाने से पहले चलाई जाती हैं।",
+    "script_fail": " ऐसा लगता है कि प्री-रिक्वेस्ट स्क्रिप्ट में गड़बड़ है। नीचे दी गई त्रुटि की जाँच करें और उसके अनुसार स्क्रिप्ट को ठीक करें।",
+    "test_script_fail": " ऐसा लगता है कि परीक्षण स्क्रिप्ट में कोई त्रुटि है। कृपया त्रुटियों को ठीक करें और फिर से परीक्षण चलाएं"
+  },
+  "mqtt": {
+    "log": " लकड़ी का लट्ठा",
+    "url": " यूआरएल"
+  },
+  "navigation": {
+    "graphql": " ग्राफक्यूएल"
+  },
+  "request": {
+    "save": " बचाना",
+    "title": " प्रार्थना",
+    "url": " यूआरएल"
+  },
+  "response": {
+    "json": " JSON"
+  },
+  "settings": {
+    "interceptor": " इंटरसेप्टर",
+    "sync_environments": " वातावरण",
+    "sync_history": " इतिहास"
+  },
+  "shortcodes": {
+    "url": " यूआरएल"
+  },
+  "shortcut": {
+    "request": {
+      "title": " प्रार्थना"
+    }
+  },
+  "socketio": {
+    "events": " आयोजन",
+    "log": " लकड़ी का लट्ठा",
+    "url": " यूआरएल"
+  },
+  "sse": {
+    "log": " लकड़ी का लट्ठा",
+    "url": " यूआरएल"
+  },
+  "state": {
+    "bulk_mode_placeholder": "प्रविष्टियां न्यूलाइन द्वारा अलग की जाती हैं कुंजी और मान इसके द्वारा अलग किए जाते हैं: जिस भी पंक्ति को आप जोड़ना चाहते हैं उसके लिए # प्रीपेन्ड करें लेकिन अक्षम रखें"
+  },
+  "tab": {
+    "history": " इतिहास",
+    "schema": " योजना"
+  },
+  "team": {
+    "invite": " आमंत्रित करना"
+  },
+  "test": {
+    "script": " लिखी हुई कहानी"
+  },
+  "websocket": {
+    "log": " लकड़ी का लट्ठा",
+    "protocols": " प्रोटोकॉल",
+    "url": " यूआरएल"
+  }
+}

```

Comment:
what about the remaining strings in source file `en.json`?

Seems incomplete. Only make the PR `Ready for review` when complete.